### DISCUSSION TITLE
SQL Views [Jul 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2674] [Discord Threads]
> 🙋‍♂️ **drasticstuff**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### If you are connecting to external DB (MySQL/SQLite), NocoDB does support & import SQL Views when you connect.    
####    
#### There is no live console to feed SQL queries currently  
#  
> 🙋‍♂️ **peter piper**  
#### for live queries, you may want to just use a db admin application that will handle it for you.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/993598995005972541)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
